src/components/context.ts

```diff
@@ -6,10 +6,18 @@ import { writable } from 'svelte/store';
 const CONTEXT_KEY = 'storybook-registration-context';
 const CONTEXT_KEY_COMPONENT = 'storybook-registration-context-component';
 
+// this is used to keep track of the arguments of the stories at render time
+export const storyArguments = new Map<string, unknown>();
```

Comment:
having a shared global states with all stories doesn't feel good to me..
moreover, the name doesn't uniquely identify a story.

I am afraid of behavior in devmode, or conflicts with the others stories